playground/package.json

```diff
@@ -9,11 +9,9 @@
     "postinstall": "nuxi prepare"
   },
   "devDependencies": {
+    "@prisma/client": "^5.21.1",
```

Comment:
should this be a dev dependency? I guess it's the playground so it doesn't matter, but seems odd.

Reply:
@jharrell It does look odd, but I'll clean this up in a followup PR. Could you approve this for now?